app/controllers/plans_controller.rb

```diff
@@ -101,7 +101,7 @@ def create
       if plan_params[:funder].present? && plan_params[:funder][:id].present?
         attrs = plan_params[:funder]
         attrs[:org_id] = attrs[:id]
-        @plan.funder = org_from_params(params_in: attrs, allow_create: false)
+        @plan.funder = org_from_params(params_in: attrs, allow_create: true)
```

Comment:
I don't think you need to change this. I tested adding funders with the current code and all worked fine.

Could you verify the behavior for the allow_create parameter and test it without modification to this part of the controller?